pkg/i18n/english.go

```diff
@@ -953,6 +954,7 @@ func EnglishTranslationSet() TranslationSet {
 		DiffingMenuTitle:                    "Diffing",
 		LcSwapDiff:                          "reverse diff direction",
 		LcOpenDiffingMenu:                   "open diff menu",
+		LcLaunchDiffTool:                    "Launch git difftool",
```

Comment:
`Lc` in `LcSomeTitle` actually denotes `lowercase`, could you change it to `launch git difftool` then?